### ISSUE TITLE
Empty arrays give error when trying to read XML

### ISSUE THREAD

**Issue by bennorth** [bennorth/python-xml-serdes#3]
As reported by a user:

[...] passing an empty array into xml-serdes [...] works when going from Python to xml, but can’t be loaded back in. It gives this error:

```
    def _extract_from(self, elt, expected_tag, _xpath):
        s_elts = elt.text.split(',')
>       elements_list = list(map(self.dtype, s_elts))
E       ValueError: could not convert string to float:
```

**Comment by bennorth** [comment]
Fixed in `v2.0.2`.

**Comment by bennorth** [comment]
As reported by the same user, the initial fix did not solve all problems.  Trying to parse the XML fragment
```xml
<lengths></lengths>
```
as a NumPy vector of (say) `int32` values throws an error.

Fixed in `v2.0.3` and explicitly tested in `v2.0.4`.
